.gitignore

```diff
@@ -1,5 +1,7 @@
 /alm
 /alm-cli
+/alm.exe
+/alm-cli.exe
```

Comment:
I'd say: remove those two lines and Instead ignore all executable files in any directory by using this pattern: `**/*.exe`. See here for more infomation: https://git-scm.com/docs/gitignore